nhais-adaptor/incoming.py

```diff
@@ -0,0 +1,34 @@
+import os
+import json
+import adaptor.incoming.operation_definition_adaptor as adaptor
+import edifact.incoming.parser.deserialiser as deserialiser
+from PyInquirer import prompt
+
+dirs = os.listdir("./mailbox/NHAIS")
+
+nhais_outbox_choices = [
+    {
+        'type': 'list',
+        'name': 'nhais',
+        'message': 'Select a NHAIS outbox to read from: ',
+        'choices': dirs
+    }
+]
+
+answers = prompt(nhais_outbox_choices)
+nhais_cypher = answers["nhais"]
+
+with open(f"./mailbox/NHAIS/{nhais_cypher}/outbox/edifact.txt", "r") as incoming_edifact_file:
+    incoming_interchange_raw = incoming_edifact_file.read()
+
+lines = incoming_interchange_raw.split("'\n")

```

Comment:
python's string has a `.splitlines()` method, might be better to use that over a specific line ending character.

Reply:
Yeah that's right. Do you guys think it ok to leave it as it is or worth investigating further?

Reply:
As discussed offline, it looks like actual Edifact output won't contain newlines anyway, so probably OK to leave this as it is in this PR.

Reply:
That's fine I missed the Edifact line ending character, it's fine to leave it I agree